src/client.ts

```diff
@@ -128,13 +136,10 @@ class DecisionClient {
           }
           let headers = context.init.headers as Record<string, string>;
           if (!!additionalOpts.includeExplanation) {
-            log('--------------------------------------------------------------');
-            log('              !!! WARNING - WARNING - WARNING !!!             ');
-            log('');
-            log('You have opted to include explainer details with this request!');
-            log('This will cause performance degradation and should not be done');
-            log('in production environments.');
-            log('--------------------------------------------------------------');
+            logger(
+              'warn',
```

Comment:
Oddly enough, we want to keep the giant ugly formatted message in this case. @karnowski specifically requested this situation be especially loud because of the extra overhead of getting explainer results.

A single call to `logger` with `\n` splitting the lines is fine though.

Reply:
Updated